cosmo_tester/resources/terraform/scripts/linux-cli-test.sh

```diff
@@ -24,20 +20,18 @@ fi
 
 set -e
 
-echo "Creating inputs file.."
-echo "public_ip: ${PUBLIC_IP}
-private_ip: ${PRIVATE_IP}
-ssh_user: ${MANAGER_USER}
-ssh_key_filename: ${PRIVATE_KEY_PATH}
-admin_username: admin
-admin_password: admin" > ${INPUTS_FILE_PATH}
-cat ${INPUTS_FILE_PATH}
+cfy profiles use ${PRIVATE_IP} -u admin -p admin -t default_tenant
+
+cfy blueprints upload https://github.com/cloudify-cosmo/cloudify-hello-world-example/archive/master.tar.gz -b bp -n singlehost-blueprint.yaml
```

Comment:
I think it would be better for us to start using a test blueprint contained with the tests themselves, but we can do that separately (just flagging it here as a 'future-thought-relevant' thing).

Reply:
Agreed. We'll have a separate PR for that.
Maybe like what @funkyHat suggested we do instead of using hello world for sanity.

Reply:
`cfy blueprints upload cloudify-cosmo/cloudify-hello-world` is nicer than the whole github link

Reply:
Oh yeah, I forgot you can do that. Fill do